scripts/installation/install-openkat.yml

```diff
@@ -0,0 +1,588 @@
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Library General Public
+# License as published by the Free Software Foundation; either
+# version 2 of the License, or (at your option) any later version.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Library General Public License for more details.
+#
+# You should have received a copy of the GNU Library General Public
+# License along with this library; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301, USA
+#
+# Changelog
+# Date       Author   Comments
+# ===========================================================================
+# 2023-08-12 J. Baten Initial version
+# 2023-08-14 J. Baten Debugging. Hacking Superadmin password to support 
+#                     current stable standard Ansible
+# 2023-09-16 J. Baten Inject variables for version numers, and prepare for MR.                    
+
+# Usage; Make sure you can ssh to the root account on the remote server using
+#        authorized_keys. After that run the following command (and don't forget
+#        the comma!
+#
+#        ansible-playbook -vvv install-openkat.yml -i <hostname or ip>, -u root
+
+---
+
+- name: Playbook to install OpenKAT 1.10.1 on Debian 12
+  hosts: all
+
+  vars:
+    katrelease: "1.11.0"
+    xtdbrelease: "1.0.7"
+
+  tasks:
+
+  # Downloading and installing
+  # Download the packages for all the components of KAT from GitHub. Also download the XTDB multinode package from GitHub.
+
+  - name: "Download release {{ katrelease }} to our Debian 12 server"
+    get_url:
+      url: "https://github.com/minvws/nl-kat-coordination/releases/download/v{{ katrelease }}/kat-debian12-{{ katrelease }}.tar.gz"
+      dest: /tmp
+      mode: '0440'
+
+  - name: "Download release xtdb {{ xtdbrelease }} to our Debian 12 server"
+    get_url:
+      url: "https://github.com/dekkers/xtdb-http-multinode/releases/download/v{{ xtdbrelease }}/xtdb-http-multinode_{{ xtdbrelease }}_all.deb"
+      dest: /tmp
+      mode: '0440'
+
+  # After downloading they can be installed as follows:
+  # tar zvxf kat-*.tar.gz
+  - name: Unpack tar OpenKAT file
+    unarchive:
+      src: "/tmp/kat-debian12-{{ katrelease }}.tar.gz"
+      remote_src: yes
+      dest: /tmp
+
+  # sudo apt install --no-install-recommends ./kat-*_amd64.deb ./xtdb-http-multinode_*_all.deb
+  - name: Install all needed packages
+    apt:
+      deb: "/tmp/{{ item }}"
+      install_recommends: false
+    loop:
+      - "kat-boefjes_{{ katrelease }}_amd64.deb"
+      - "kat-bytes_{{ katrelease }}_amd64.deb"
+      - "kat-keiko_{{ katrelease }}_amd64.deb"
+      - "kat-mula_{{ katrelease }}_amd64.deb"
+      - "kat-octopoes_{{ katrelease }}_amd64.deb"
+      - "kat-rocky_{{ katrelease }}_amd64.deb"
```

Comment:
Arch 'amd64' hier mogelijk ook in een variabele stoppen... uiteindelijk zullen er vast ook arm64 packages komen.

Reply:
Vast wel. Ooit. En Risc-V natuurlijk.

Reply:
Er is zeker een doel om bepaalde packages ook voor Raspi's beschikbaar te gaan maken. Dat moet ook geen jaren duren.

Reply:
Vooral ook voor arm-clouds .... dat zijn goede platforms voor tooling zoals dit. Ik draai 90% van mijn services momenteel op arm-servers (hetzner cloud)